StripesFormWrapper.js

```diff
@@ -51,7 +51,10 @@ class StripesFormWrapper extends Component {
 
   continue() {
     this.unblock();
-    this.props.history.push(this.state.nextLocation.pathname);
+    this.props.history.goBack();
```

Comment:
How on earth does this work?!

Reply:
Oh, OK -- you described it in the header.

Well, this is unspeakable. Ph'nglui mglw'nafh Cthulhu R'lyeh wgah'nagl fhtagn.

Reply:
Haha that looks like icelandic language to me :)

Reply:
You don't recognise it? It's pretty famous :-)

Reply:
I had to take a closer look again ;) https://en.wikipedia.org/wiki/Cthulhu